zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -622,34 +621,34 @@ private long getUnixTimeLastRunParagraph(Paragraph paragraph) {
         gotNoteId = note.getId();
       }
     }
-    return getJobListBymNotebookId(gotNoteId);
+    return getJobListByNoteId(gotNoteId);
   }
 
-  public List<Map<String, Object>> getJobListBymNotebookId(String notebookID) {
+  public List<Map<String, Object>> getJobListByNoteId(String noteID) {
```

Comment:
Could you make it to use `Id` instead of `ID` for consistency? We have camel convention.